guides/v2.3/magento-functional-testing-framework/2.3/commands/mftf.md

```diff
@@ -235,23 +252,26 @@ Generates and executes the listed groups of tests using Codeception.
 
 #### Usage
 
-`run:group [options --] <group1> <group2> ...`
+```bash
+vendor/bin/mftf run:group [--skip-generate|--remove] [--] <group1> [<group2>] 
+```
 
 #### Options
 
 Option | Description
 ---|---
 `-k, --skip-generate` | Skips generating from the source XML. Instead, the command executes previously-generated groups of tests.
+`-r, --remove`   | Removes previously generated suites and tests before the actual generation and run.
 
 #### Examples
 
-Generate from XML and execute the tests with the annotations `group="group1"` and `group="group2"`:
+Clean up after the last run, generate from XML and execute the tests with the annotations `group="group1"` and `group="group2"`:
```

Comment:
I agree if it would be a conceptual topic or a tutorial where a reader is supposed to read the entire topic. This topic is mostly a reference, that's why it contains the full explanation in each command description.